examples/pennylane_QGAN.py

```diff
@@ -0,0 +1,224 @@
+"""
```

Comment:
The following were all generated from the previous `examples/*.ipynb` files using the Jupyter conversion tools.